src/app/containers/calculator/calculator.container.ts

```diff
@@ -85,31 +73,30 @@ export class CalculatorComponent implements OnInit {
         }
     };
     results: {};
+    sessionId: string = 'id-' + Date.now();
```

Comment:
```suggestion
    sessionId: string = this.newSessionId();
```
Can you call the method you created in the initializer here too?

Reply:
Doesn't work since `this.newSessionId()` does not return anything, just assigns to `this.sessionId`. 

If you prefer, I could initialize the component with sessionId undefined and call this.newSessionId() immediately. I don't have a preference (but I guess if we change the prefix, the way it is now it will need to be updated in two places). I will update to the other way to initialize, so prefix is only in one place.